Objects/rangeobject.c

```diff
@@ -71,43 +72,35 @@ make_range_object(PyTypeObject *type, PyObject *start,
    range(0, 5, -1)
 */
 static PyObject *
-range_new(PyTypeObject *type, PyObject *args, PyObject *kw)
+range_from_array(PyTypeObject *type, PyObject *const *args, size_t num_args)
```

Comment:
I would prefer to use Py_ssize_t type for num_args. Usually, it's called "nargs", but I'm fine with "num_args" ;-)

Reply:
Right, `Py_ssize_t` is more consistent.
But I find `num_args` more descriptive, so I'll keep that name.